interface/forms/soap/templates/general_new.html

```diff
@@ -4,49 +4,79 @@
 <title>{xl t='SOAP'|escape:'html'}</title>
 
 {headerTemplate}
+{literal}
+ <style type="text/css" title="mystyles" media="all">
+ <!--
```

Comment:
No, This internal stylesheet gives the background color for the legend and fieldset for all non flat stylesheets. You will find this internal stylesheet in all the forms that I have modified.  Once a reasonable number of pages have been restyled this stylesheet along with any other modification can be made into an openemr specific override of the default bootstrap styles.
The html style block comment is invalid in css.